docs/governance/special-interest-group-process.md

```diff
@@ -0,0 +1,23 @@
+# Special Interest Group
+
+A _special interest group (SIG)_ under the Technical Advisory Council (TAC) is a group with a shared interest in advancing a specific area of knowledge, learning, or technology related to the mission of the OpenWallet Foundation where members cooperate to affect or to produce solutions within their particular field. Unlike a [task force](./task-force-process.md), SIGs are typically long running and may or may not produce any deliverables. A SIG can be created by anyone in the OpenWallet Foundation community and must be proposed to and approved by the TAC.
```

Comment:
Should we change "long running" to "long-running"?

Reply:
I checked a couple of different grammar sites, and it looks like `long running` is correct in this instance. I believe that if I changed the text to "SIGs are typically long-running groups", then it would require the hyphen.